ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-master.yaml

```diff
@@ -0,0 +1,45 @@
+canonical_go_repository: github.com/kubevirt/hyperconverged-cluster-operator
```

Comment:
does the code for your test image also live in this repository?

if so, you may be able to do something like the console team did here in which they get the "src" image (which is just the build_root image plus your the git repo/PR branch added on top) re-tagged as console-tests and promoted into the ocp namespace so they can then reference it from their CI jobs:

https://github.com/openshift/release/blob/f6421918f8eeff6f7f0e427bb84bc829f95a99c2/ci-operator/config/openshift/console/openshift-console-master.yaml#L21-L25

that image then gets used by tests like this one:
https://github.com/openshift/release/blob/f6421918f8eeff6f7f0e427bb84bc829f95a99c2/ci-operator/config/openshift/cluster-authentication-operator/openshift-cluster-authentication-operator-master.yaml#L48-L52

which uses the console-installer template which runs commands against the console-tests image:
https://github.com/openshift/release/blob/f6421918f8eeff6f7f0e427bb84bc829f95a99c2/ci-operator/templates/openshift/installer/cluster-launch-installer-console.yaml#L18-L20

https://github.com/openshift/release/blob/f6421918f8eeff6f7f0e427bb84bc829f95a99c2/ci-operator/templates/openshift/installer/cluster-launch-installer-console.yaml#L92

($TEST_COMMAND will be `./test-prow-e2e.sh login` in this case).

The work i've been doing is to allow you to implement a similar flow in which instead of a hardcoded `console-tests` you can instead reference an arbitray test image:
https://github.com/openshift/release/blob/5976527b2e6a07f898b54eded83e846022485aff/ci-operator/config/openshift/cluster-authentication-operator/openshift-cluster-authentication-operator-master.yaml#L48-L53

Reply:
(you can largely ignore what i said above, if you're only concerned w/ running this job against PRs against your own repository....if that is all you care about, then what you've done, using openshift_installer_src, is fine)

Reply:
I think that I definitely want to explore that as a next step. The tests we would want to run are actually distributed across different images from different kubevirt repos. Not sure, if and how we can pull them in.

As a first step it is already great for us, if we can verify on PRs on the hyperconverged-cluster-operator, that it can successfully install all our operators (what this PR does).

Reply:
(you will need to do the promotion bit in order to fix up your periodic job in the other PR anyway, because your periodic job can't run it if you're not promoting it).

Reply:
Makes sense. Should I add this immediately or in a follow up PR?

Reply:
might as well do it now.

Reply:
Done.

Reply:
The src image is now also promoted.